eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageCodecs.scala

```diff
@@ -298,7 +298,7 @@ object LightningMessageCodecs {
 
   val unknownMessageCodec: Codec[UnknownMessage] = (
     ("tag" | uint16) ::
-      ("message" | varsizebinarydata)
+      ("message" | variableSizeBits(uint16, bits))
```

Comment:
Except this probably reduces max size of message by 8? Should `uint24` be used here?